README.md

```diff
@@ -1,173 +1,250 @@
+
+
+## Understanding the deployed components
+
+TODO
+
+## Development / contributing
+
+Please refer to [this documentation](./DEVELOPMENT.md) for everything related to building, deploying or bundling from sources.
 
-## Publish on central OperatorHub
+## F.A.Q / Troubleshooting
 
-See [RELEASE.md](./RELEASE.md#publishing-on-operatorhub).
+If you can't find help here, don't hesitate to open [an issue](https://github.com/netobserv/network-observability-operator/issues) or a [Q&A](https://github.com/netobserv/network-observability-operator/discussions/categories/q-a). There are several repositories under _netobserv_ github org, but it is fine to centralize these in _network-observability-operator_.
 
-## FlowCollector custom resource
+### Is it for OpenShift only?
 
-The `FlowCollector` custom resource is used to configure the operator and its managed components. You can read its [full documentation](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowCollector.md) and check this [sample file](./config/samples/flows_v1alpha1_flowcollector.yaml) that you can copy, edit and install.
+No! While some features are developed primarily for OpenShift, we want to keep it on track with other / "vanilla" Kubes. For instance, there has been some work to make the console plugin [run as a standalone](https://github.com/netobserv/network-observability-console-plugin/pull/163), or the operator to manage upstream (non-OpenShift) [ovn-kubernetes](https://github.com/netobserv/network-observability-operator/pull/97).
 
-Note that the `FlowCollector` resource must be unique and must be named `cluster`. It applies to the whole cluster.
+And if something is not working as hoped with your setup, you are welcome to contribute to the project ;-)
 
-## Enabling OVS IPFIX export
+### Which version of Kubernetes / OpenShift is supported?
 
-If you use OpenShift 4.10 or the upstream ovn-kubernetes without OpenShift, you don't have anything to do: the operator will configure OVS *via* OpenShift Cluster Network Operator, or the ovn-kubernetes layer directly.
+It depends on which `agent` you want to use: `ebpf` or `ipfix`, and whether you want to get the OpenShift Console plugin.
 
-Else if you use OpenShift 4.8 or 4.9, some manual steps are still required
+#### To run the eBPF agent
+
+What matters is the version of the Linux kernel.
+TODO: precisions
+Other than that, there are no known restrictions (yet?) on the Kubernetes version.
+
+#### To use IPFIX exports
+
+OpenShift 4.10 or above, or upstream OVN-Kubernetes [TODO: upstream version?] are recommended, as the operator will configure OVS for you. Else, you need to configure it manually.
```

Comment:
@mariomac I didn't find how upstream ovn-k is versionned, do you know? their github doesn't tell much about it (or I missed it)